ding_dibs.install

```diff
@@ -30,6 +67,24 @@ function ding_dibs_install() {
     'default' => 0,
     'description' => '0 = capture failed, 1 = capture succeeded',
   ));
+  variable_set('ding_dibs_start_cron', 2);
+  drupal_install_schema('ding_dibs');
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function ding_dibs_uninstall() {
+  $variables = array(
+    'ding_dibs_start_cron',
+    'ding_dibs_start_operation_cron',
+    'ding_dibs_start_cron_minute',
+    'ding_dibs_emails_validate',
+  );
+  for ($start = 0; $start < count($variables); $start++) {
```

Comment:
`$start` is not a suitable name for an iterator variable. Consider using `foreach` here instead.